backend/features/support/greating.steps.mjs

```diff
@@ -0,0 +1,13 @@
+import assert from 'assert';
```

Comment:
let's delete this test (and it's related class), as it's not directly used/related to the project